x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/get_columns.tsx

```diff
@@ -36,9 +36,20 @@ const appToIcon = (app: string) => {
   if (app === 'dashboards') {
     return 'dashboard';
   }
+  if (app === 'ml') {
+    return 'machineLearning';
+  }
+
   return app;
 };
 
+// Helper function: translate an app id to user friendly string
+const appToTooltip = (appId: string | undefined) => {
+  if (appId === 'ml') {
+    return 'Machine Learning';
```

Comment:
This is probably okay for now, but we need to make sure these are translated correctly.